multi_time_trajectory_controller/src/multi_time_trajectory_controller.cpp

```
@@ -66,7 +66,8 @@ controller_interface::CallbackReturn MultiTimeTrajectoryController::on_init()
     params_ = param_listener_->get_params();
 
     joint_limiter_loader_ = std::make_shared<pluginlib::ClassLoader<JointLimiter>>(
-      "joint_limits", "joint_limits::JointLimiterInterface<joint_limits::JointLimits>");
+      "joint_limits",
+      "joint_limits::JointLimiterInterface<trajectory_msgs::msg::JointTrajectoryPoint>");
```

Should we throw an exception or at least print an error if it can't load the plugins. At least then, we catch this earlier.

---

This does throw an error, that's how I ended up catching it. The tests are generally verbose so it might have gotten lost in the messages, but in a regular system run it should be more obvious.

---

Definitely don't see an error when running the software stack

---

If it's set to the wrong joint limiter it doesn't throw an error? I thought we aren't using a joint limiter at the moment with the MAC

---

We aren't using the joint limiter in the MAC. I was just wondering if it we should be reporting an error if it doesn't find any plugin as before this fix, it wasn't finding any plugins

---

Ah I see. I think if it doesn't find one it's fine, since it doesn't have to use one. It should only throw an error if one's given but the one given is invalid, I think

---

OK. I guess that is fine for now. I was only concerned about not finding interface changes to the joint limiter till later.